app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedFragmentPresenter.kt

```diff
@@ -67,22 +65,22 @@ class RecentlyPlayedFragmentPresenter @Inject constructor(
   }
 
   private val ongoingStoryListSummaryResultLiveData:
-    LiveData<AsyncResult<OngoingStoryList>>
+    LiveData<AsyncResult<PromotedActivityList>>
     by lazy {
-      topicListController.getOngoingStoryList(
+      topicListController.getPromotedActivityList(
         ProfileId.newBuilder().setInternalId(internalProfileId).build()
       ).toLiveData()
     }
 
   private fun subscribeToOngoingStoryList() {
-    getAssumedSuccessfulOngoingStoryList().observe(
+    getAssumedSuccessfulPromotedActivityList().observe(
       fragment,
-      Observer<OngoingStoryList> { it ->
-        if (it.recentStoryCount > 0) {
+      Observer<PromotedActivityList> { it ->
+        if (it.promotedStoryList.recentlyPlayedStoryList.isNotEmpty()) {
           val recentSectionTitleViewModel =
             SectionTitleViewModel(activity.getString(R.string.ongoing_story_last_week), false)
           itemList.add(recentSectionTitleViewModel)
-          for (promotedStory in it.recentStoryList) {
+          for (promotedStory in it.promotedStoryList.recentlyPlayedStoryList) {
             val ongoingStoryViewModel =
               OngoingStoryViewModel(
```

Comment:
Also might be useful to name this `PromotedActivityViewModel` since this includes suggested stories now, too.